tests/misc_env/sosreport.py

```diff
@@ -47,7 +47,7 @@ def generate_sosreport(cnode, results):
         cnode.exec_command(cmd="sudo yum -y install sos", check_ec=False)
 
         out, err = cnode.exec_command(
-            sudo=True, cmd="sosreport -a --all-logs -e ceph --batch", timeout=1200
+            sudo=True, cmd="sos report -a --all-logs -e ceph --batch", timeout=1200
```

Comment:
@rkhambad , If it is tested with above changes could you please attach pass logs

Reply:
i am really sorry, i need to make this as draft PR to just check, if it affect anything. And i forget to make that and it went open for review.